src/lib/tls/tls_callbacks.h

```diff
@@ -560,6 +560,26 @@ class BOTAN_PUBLIC_API(2, 0) Callbacks {
       virtual void tls_log_debug_bin(const char* descr, const uint8_t val[], size_t val_len) {
          BOTAN_UNUSED(descr, val, val_len);
       }
+
+      /**
+       * Optional callback: Allows access to a connection's secret data
+       *
```

Comment:
This should also note that these functions are only invoked if `Policy::allow_ssl_key_log_file` returns true.